library/src/main/java/tiamat/RxSharedPreferences.java

```diff
@@ -6,42 +6,44 @@
 import java.util.Map;
 import java.util.Set;
 
-import rx.Observable;
-import rx.Subscriber;
-import rx.functions.Action0;
-import rx.subscriptions.Subscriptions;
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.FlowableEmitter;
+import io.reactivex.FlowableOnSubscribe;
+import io.reactivex.functions.Cancellable;
+
 
 public abstract class RxSharedPreferences {
 
     private final SharedPreferences preferences;
-    private final Observable<String> changeEvents;
+    private final Flowable<String> changeEvents;
 
     protected RxSharedPreferences(Context context, String name) {
         this(context.getSharedPreferences(name, Context.MODE_PRIVATE));
     }
 
     protected RxSharedPreferences(SharedPreferences preferences) {
         this.preferences = preferences;
-        this.changeEvents = Observable.create(new Observable.OnSubscribe<String>() {
+        this.changeEvents = Flowable.create(new FlowableOnSubscribe<String>() {
             @Override
-            public void call(final Subscriber<? super String> subscriber) {
+            public void subscribe(final FlowableEmitter<String> flowableEmitter) throws Exception {
                 final SharedPreferences.OnSharedPreferenceChangeListener listener = new SharedPreferences.OnSharedPreferenceChangeListener() {
                     @Override
-                    public void onSharedPreferenceChanged(SharedPreferences preferences, String key) {
-                        subscriber.onNext(key);
+                    public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+                        flowableEmitter.onNext(key);
                     }
                 };
 
                 RxSharedPreferences.this.preferences.registerOnSharedPreferenceChangeListener(listener);
 
-                subscriber.add(Subscriptions.create(new Action0() {
+                flowableEmitter.setCancellable(new Cancellable() {
                     @Override
-                    public void call() {
+                    public void cancel() throws Exception {
                         RxSharedPreferences.this.preferences.unregisterOnSharedPreferenceChangeListener(listener);
                     }
-                }));
+                });
             }
-        }).share();
+        }, BackpressureStrategy.LATEST).share();
```

Comment:
Well, I'm not sure in this case we need `Backpressure` actually...isn't `Observable.create` enough? What do you think?😃